code/modules/mob/living/carbon/brain/brain.dm

```diff
@@ -5,7 +5,7 @@
 	var/obj/item/container = null
 	var/timeofhostdeath = 0
 	var/emp_damage = 0//Handles a type of MMI damage
-	var/alert = null

```

Comment:
unused now since I simplified the brain's emp effects and it was only used there.